src/test/java/io/appium/java_client/touch/TouchOptionsTests.java

```diff
@@ -60,7 +60,7 @@ public void longPressOptionsShouldBuildProperly() {
                 .withDuration(ofMillis(1))
                 .build();
         final Map<String, Object> expectedOpts = new HashMap<>();
-        expectedOpts.put("element", ((HasIdentity) DUMMY_ELEMENT).getId());
+        expectedOpts.put("element", ((RemoteWebElement) DUMMY_ELEMENT).getId());
```

Comment:
such cast is probably redundant if DUMMY_ELEMENT is already RemoteWebElement

Reply:
DUMMY_ELEMENT is declared as:
`private static final WebElement DUMMY_ELEMENT = new DummyElement();`

I can change variable type to `RemoteWebElement` or `DummyElement`, but I need to revise the logic of other tests, because it could be done intentionally to cover some logic.

Reply:
`io.appium.java_client.touch.offset.ElementOption#withElement` contains type check and type casting logic, so it looks like such declaration is a way to test the method logic.